packages/components/src/z-stack/component.tsx

```diff
@@ -20,9 +16,7 @@ import { getValidChildren } from '../ui/utils/get-valid-children';
 import { contextConnect, useContextSystem } from '../ui/context';
 // eslint-disable-next-line no-duplicate-imports
 import type { PolymorphicComponentProps } from '../ui/context';
-import { View } from '../view';
```

Comment:
Are we tracking the removal of the `View` component (and the `createComponent` function) anywhere?

Reply:
I think it will be removed as just a side effect of this work, I'm not sure it's necessary to track it separately.